src/seedu/addressbook/util/FilePathUtil.java

```diff
@@ -0,0 +1,66 @@
+package seedu.addressbook.util;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * This is a utility class for file path validation.
+ */
+public class FilePathUtil {
+    private static final int EXTENSION_INVALID_INDEX = 0;
+
+    /**
+     * Returns true if the given file is acceptable.
+     * The file path is acceptable if it does not contain any reserved characters <>:"/|?*,
+     * has a filename with an extension.
+     * TODO: Implement a more rigorous validity checking.
+     */
+    public static boolean isValidFilePath(String filePath) {
+        return isValidPath(filePath) && isValidFileName(filePath);
```

Comment:
This split into two methods seems a bit artificial unless you split the parameter into path and filename first. See if you can merge the two methods back.
Alternatively, the logic can be like this:
* extract file name
* check file name to see if it acceptable
* check remainder to see if it is a folder exists